src/plugins/aws/ssm/index.ts

```diff
@@ -516,10 +542,13 @@ export const scp = async (
     // This_might be overkill because we are already spawning a subprocess that will run the commands for us
     // but just in case someone enters that subprocess we're also disabling the history of commands run.
     `unset HISTFILE`,
-    `eval $(ssh-agent) >/dev/null 2>&1`,
+    // in debug mode, we want to see the pid of the ssh-agent and compare it to the environment variable
+    `eval $(ssh-agent)${args.debug ? "" : " >/dev/null 2>&1"}`,
     `trap 'kill $SSH_AGENT_PID' EXIT`,
     `ssh-add -q - <<< '${privateKey}'`,
-    createScpCommand(data, args).join(" "),
+    // in debug mode, we'll see the keys that were added to the agent and more information about the agent
+    ...(args.debug ? debug : []),
+    scpCommand,
     `SCP_EXIT_CODE=$?`,
     `exit $SCP_EXIT_CODE`,
   ];
```

Comment:
Suggestion: also print the sequence of commands that can be used to create a similar SCP flow. I tested this and seems to work.

```suggestion
  ];

  // Print commands that can be individually executed to reproduce behavior
  // Remove the debug information - can be executed manually between steps
  const reproCommands = without([
    "bash",
    ...Object.entries(process.env).map(([key, value]) => `export ${key}='${value}'`),
    ...Object.entries(credential).map(([key, value]) => `export ${key}='${value}'`),
    ...writeStdin
  ], ...debug);
  print2(`Execute the following commands to create a similar SCP session:\n *** COMMANDS BEGIN ***\n${reproCommands.join("\n")}\n *** COMMANDS END ***`);
```